TOC.md

```diff
@@ -68,7 +68,7 @@
     + Use BR Tool (Recommended)
       + [Use BR Tool](/br/backup-and-restore-tool.md)
       + [BR Use Cases](/br/backup-and-restore-use-cases.md)
-      + [BR storages](/br/backup-and-restore-storages.md)
+      + [BR Storages](/br/backup-and-restore-storages.md)
```

Comment:
Please also help correct the capitalization for the title of `backup-and-restore-storages.md`.

Reply:
Done in https://github.com/pingcap/docs/pull/4214/commits/21eb650925006cb156ffba671043d288bb5dceea.